app/src/main/java/org/simple/clinic/teleconsultlog/success/TeleConsultSuccessInit.kt

```diff
@@ -0,0 +1,13 @@
+package org.simple.clinic.teleconsultlog.success
+
+import com.spotify.mobius.First
+import com.spotify.mobius.Init
+import org.simple.clinic.mobius.first
+import org.simple.clinic.teleconsultlog.success.TeleConsultSuccessEffect.LoadPatientDetails
+
+class TeleConsultSuccessInit : Init<TeleConsultSuccessModel, TeleConsultSuccessEffect> {
+  override fun init(model: TeleConsultSuccessModel): First<TeleConsultSuccessModel, TeleConsultSuccessEffect> {
+    return first(model, LoadPatientDetails(model.patientUuid))
```

Comment:
Not necessary to do it in this PR, but we should split this into two commits:

- One where we add the effect and its handler
- The next where we trigger the effect from the init function

Reply:
Question on 1. 
So the event translated as a response from the effect handler should also be in the same commit?

Reply:
Yes, that should be in the same commit. This is more about making commits atomic and easier to review than anything else since we squash and merge PRs anyway.